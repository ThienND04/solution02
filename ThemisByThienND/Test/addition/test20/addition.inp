10
1807 1249 1073 443658
1930 1272 1544 850878
1923 1709 1440 438165
1492 1042 1987 922503
1327 1729 1840 542612
1303 1169 1709 197157
1560 1933 1099 380278
1816 1335 1097 707826
1512 1267 1810 277633
1979 1149 1579 758821
