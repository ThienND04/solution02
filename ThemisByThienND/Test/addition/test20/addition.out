42
165
22
139
36
8
23
95
11
82
